next-cloudinary/src/types/player.ts

```diff
@@ -17,6 +17,46 @@ export interface CloudinaryVideoPlayerOptions {
   sourceTypes?: Array<string>;
   logoImageUrl?: string;
   logoOnclickUrl?: string;
+  aiHighlightsGraph?: boolean;
```

Comment:
i know this is a bit nitpicky, but could you update the props to be in ABC order? or if you think it makes sense to group them based on how they are in the docs, you could alternatively add grouping comments similar to: 

https://github.com/colbyfayock/next-cloudinary/blob/086733de3bbc70b70d0e4577a0259c78f345c0b2/next-cloudinary/src/components/CldUploadWidget/CldUploadWidget.types.ts#L98

Reply:
ohh okay, i think it actually makes sense to group them base on how they are in the docs so i will go with that option